### PR TITLE
added docker to project

### PR DESCRIPTION
**Pull request by dmarmul**

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -29,3 +29,33 @@ databaseChangeLog:
...
+#  - include:
+#      file: db/changelog/changes/06-insert-admin-to-users-table.yaml
+#  - include:
+#      file: db/changelog/changes/07-rename-name-column-in-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/08-rename-users-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/09-insert-values-to-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/12-insert-admin-to-users-roles-table.yaml
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: Let's remove the commented code. Also, make all file names consistent;)

File: src/main/resources/db/changelog/changes/24-insert-book-to-books-table.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-book-to-books-table
+      author: mateacademystudent
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/changes/24-insert-book-to-books-table.yaml]: better use your name

File: src/main/resources/db/changelog/changes/003-create-roles-table.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/changes/003-create-roles-table.yaml]: It looks strange that you created a table with the wrong column name and renamed it in one PR. Can you create a table with the correct name?

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! Let’s  fix a few mistakes ;)

**Review by Sanderfuck — CHANGES_REQUESTED** [review]
you have changes that nor related with docker

**Comment by dmarmul** [comment]
> you have changes that nor related with docker

Can you explain?

**Pull request merged**